lib/base-compiler.js

```diff
@@ -355,6 +369,14 @@ Compile.prototype.couldSupportASTDump = function (options, version) {
     return false;
 };
 
+Compile.prototype.supportCFGGeneration = function(compilerVersion){
+    return (compilerVersion.indexOf("g++") === 0);
```

Comment:
We can probably come up with a more sophisticated check here, but this is fine for now. One thought: why is this GCC specific currently?

Reply:
sorry about that, put first thing that came to my mind.